<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IssuesView</class>
 <widget class="QWidget" name="IssuesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="searchLine">
     <property name="placeholderText">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="issuesView">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="checkButton">
     <property name="text">
      <string>&amp;Check for issues (expensive!)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
